### PR TITLE
SEN67-129: [EXT-726] Ext JS Examples : Clone and Update Coworkee example (sencha cmd version)

### PR DESCRIPTION
**Pull request by unnikrishnan-narayanan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Production build:
 Note: by default, `npm start` will use the **development** build. To run the production
 build, use the following command instead:
 
-    $ npm start -- --client-environment=production
+    $ npm start --client-environment=production
```
Comment (dalley) [inline README.md]: This looks like a bug, usually the double `--` mean pass the next argument to the actual script that is referenced from the start script.

File: client/workspace.json [hunk 2]
```diff
@@ -21,11 +21,12 @@
      *      }
      *
      */
-    "frameworks": {
-        "ext": {
+    "frameworks": {        "ext": {
```
Comment (dalley) [inline client/workspace.json]: Looks like a lint issue.

File: client/workspace.json [hunk 3]
```diff
@@ -21,11 +21,12 @@
...
             "path":"ext",
-            "version":"6.5.1"
+            "version":"6.6.0.258"
```
Comment (dalley) [inline client/workspace.json]: This will need to be 6.7.0

File: server/models/action.js [hunk 4]
```diff
@@ -26,43 +26,41 @@ module.exports = function(sequelize, DataTypes) {
...
+                }]
+            }]
+        });
+    };
+
+    Model.subject = function(action, recipient) {
+        switch (action) {
+        case 'phone':
+            var extension = recipient.get('extension');
+            return recipient.get('phone') + (extension? ':' + extension : '');
```
Comment (dalley) [inline server/models/action.js]: some minor lint issues.

File: client/workspace.json (hunk 3 above)
Comment (dalley) [reply client/workspace.json]: where you able to use 6.7.0 as the version here?

Comment (unnikrishnan-narayanan) [reply client/workspace.json]: not yet, which is why I haven't resolved this. will try today.

**Pull request merged**